test.js

```diff
@@ -24,13 +24,13 @@ describe('metalsmith collections paginate', function () {
     var metadata = {
       collections: {
         articles: [
-          { contents: '' },
-          { contents: '' },
-          { contents: '' },
-          { contents: '' },
-          { contents: '' },
-          { contents: '' },
-          { contents: '' }
+          {contents: ''},
+          {contents: ''},
+          {contents: ''},
+          {contents: ''},
+          {contents: ''},
+          {contents: ''},
+          {contents: ''}
```

Comment:
Can you please avoid unnecessary code changes. It was written that way for a reason and is not a mistake.